src/ansys/aedt/toolkits/antenna/backend/api.py

```diff
@@ -163,7 +164,7 @@ def get_antenna(self, antenna, synth_only=False):
         return antenna_parameters
 
     def export_hfss_model(self):
-        """Export model in .obj format.
+        """Export model in the OBJ format.
```

Comment:
@Samuelopez-ansys The method signature shows no parameters but there is a Parameters section. Is this perhaps a copy-and-paste error? Note that if is correct, for the ``antenna`` parameter, there is a ``:class:`` role with no link. This is true in many places, but I thought I'd point it out here.